ballerina-shell/modules/shell-cli/src/main/java/io/ballerina/shell/cli/handlers/HelpCommand.java

```diff
@@ -34,11 +34,19 @@
  * @since 2.0.0
  */
 public class HelpCommand extends AbstractCommand {
-    private final HelpProvider helpProvider;
+
+    private static final String URL = "https://ballerina.io/learn/by-example/";
+    private static final String TAGS = "<br/>";
+    private static final String EMPTY_STRING = "";
+    private static final String NEW_LINE = "\n";
+    private static final String DESCRIPTION_PREFIX = NEW_LINE + "Topic description :";
+    private static final String URL_PREFIX = NEW_LINE + "For examples visit : ";
+
+    BbeHelpProvider bbeHelpProvider;
```

Comment:
Any particular reason for this to have default visibility?

Reply:
Fixed in a1286fe5ba1e38ae8718140c18a16fadd90542e6